### PR TITLE
Upload extracted codebook indexes

### PR DESCRIPTION
**Pull request by yaozengwei**
This PR is about pruned_transducer_stateless6. It aims to allow people directly download the extracted codebook indexes for model distillation.

### CODE REVIEW

**Comment by yaozengwei** [comment]
The detailed steps are in `distillation_with_hubert.sh`.
To directly download the extracted codebook indexes for model distillation, you can set stage=2, stop_stage=4, use_extracted_codebook=True.
To start from scratch, you can set stage=0, stop_stage=4, use_extracted_codebook=False.

**Comment by pzelasko** [comment]
I see you fixed the same thing as I in #430; can you also update the HUBERT download URL? Then we could close my PR to avoid conflicts.

**Comment by yaozengwei** [comment]
> I see you fixed the same thing as I in #430; can you also update the HUBERT download URL? Then we could close my PR to avoid conflicts.

Sure. I have also updated the HUBERT download URL. See https://github.com/k2-fsa/icefall/blob/10ea04ae807b0f2eeb5d5039269e6e2f91a137cc/egs/librispeech/ASR/distillation_with_hubert.sh#L99

**Comment by pzelasko** [comment]
Thanks!

File: egs/librispeech/ASR/distillation_with_hubert.sh [hunk 1]
```diff
@@ -33,10 +35,35 @@ stage=$1
...
+
+# use_extracted_codebook can be "True" or "False"
+#   "True" -> stage 0 and stage 1 would be skipped,
+#     and directly download the extracted codebook indexes for distillation
+#   "False" -> start from scratch
+use_extracted_codebook=True
+
+# teacher_model_id can be one of
+#   "hubert_xtralarge_ll60k_finetune_ls960" -> fine-tuned model, it is the one we currently use.
+#   "hubert_xtralarge_ll60k.pt" -> pretrained model without fintuing
```
Comment (glynpu) [inline egs/librispeech/ASR/distillation_with_hubert.sh:56]: Maybe should remove the ".pt" suffix of "hubert_xtralarge_ll60k.pt"?

**Pull request merged**